src/test/resources/application.conf

```diff
@@ -1,5 +1,5 @@
 akka {
-  loglevel = INFO
+  loglevel = OFF
```

Comment:
This is the concerning change that came with this fix - see https://broadinstitute.atlassian.net/browse/GAWB-401